docs/connector-development/tutorials/cdk-tutorial-python-http/connection-checking.md

```diff
@@ -4,7 +4,7 @@ The second operation in the Airbyte Protocol that we'll implement is the `check`
 
 This operation verifies that the input configuration supplied by the user can be used to connect to the underlying data source. Note that this user-supplied configuration has the values described in the `spec.yaml` filled in. In other words if the `spec.yaml` said that the source requires a `username` and `password` the config object might be `{ "username": "airbyte", "password": "password123" }`. You should then implement something that returns a json object reporting, given the credentials in the config, whether we were able to connect to the source.
 
-In order to make requests the API, we need to specify the access 
+In order to make requests to the API, we need to specify the access. 
```

Comment:
I'm generally not entirely sure what this line is supposed to say, it feels like the 2nd half of the sentence is missing. But after this change at least it does not look like this anymore:
![image](https://user-images.githubusercontent.com/49005401/191723839-298bbc1a-d44d-4144-8efa-af10fe8a58d9.png)